woodwork/statistics_utils/_get_describe_dict.py

```diff
@@ -166,16 +174,19 @@ def _get_describe_dict(
                     values["top_values"] = []
                 else:
                     values["histogram"] = _get_histogram_values(series, bins=bins)
-                    _range = range(int(values["min"]), int(values["max"]) + 1)
-                    # Calculate top numeric values if range of values present
-                    # is less than or equal number of histogram bins and series
-                    # contains only integer values
-                    range_len = int(values["max"]) + 1 - int(values["min"])
-                    if range_len <= bins and (series % 1 == 0).all():
-                        values["top_values"] = _get_numeric_value_counts_in_range(
-                            series,
-                            _range,
-                        )
+                    if isinstance(
+                        column.logical_type,
+                        (Age, AgeNullable, Integer, IntegerNullable),
+                    ):
```

Comment:
Not something we need in this PR, but might be nice if we had an `is_integer_type` property on the column schema to use instead of having to maintain a list of logical types here.